community/cypher/cypher-planner/src/test/scala/org/neo4j/cypher/internal/compiler/planner/logical/OptionalMatchRemoverTest.scala

```diff
@@ -358,5 +360,5 @@ assert_that(
     StatementConverters.toPlannerQuery(ast.asInstanceOf[Query], table)
   }
 
-  private def parseForRewriting(queryText: String) = parser.parse(queryText.replace("\r\n", "\n"), Neo4jCypherExceptionFactory(queryText, None))
+  private def parseForRewriting(queryText: String) = parser.parse(queryText.replace(Properties.lineSeparator, "\n"), Neo4jCypherExceptionFactory(queryText, None))
```

Comment:
All these `replace` calls transforms the strings to consistently use `\n`. They should keep replacing `\r\n` strings. Otherwise on other platforms you turn this into replacing `\n` with `\n`.

Reply:
I understand, but there should not be a case of *nix system to have `\r\n`, and replacing `\n` with `\n` would do nothing (which is not harmful).

The Git on Windows by default makes the file (so the string) with system-specific EOL.

Reply:
I would argue that replacing `\r\n\` with `\n` on *nix system is not harmful either. So I think we should leave it as is.

Reply:
Well, it should be changed:

- Mainly to make the tests pass on other systems which uses `\r` only (Mac OS)
- To be uniform with newly added changes, otherwise we should also change all places to `replace("\r\n", "\n")`, ignoring Mac OS.

Reply:
Mac OS has not used `\r` since 2001 so I think we are safe :wink:

Reply:
Thanks for info.

Change reverted.

All other places were changed to `.replace("\r\n", "\n")`